### DISCUSSION TITLE
Are File/Folder Bookmark supported on MacOS ?

### DISCUSSION THREAD

**Discussion by HEIC-to-JPEG-Dev** [AvaloniaUI/Avalonia#16870] [Q&A]
The documentation is contradictory and causing confusion.

On "StorageProvier" - Bookmarks are not properly supported on desktop platforms and instead return file path instead. **MacOS support is planned** in order to get it work with Sandboxed Apple Store apps.

On the Bookmarks page it mentions nothing of this and suggests it is supported.

so, on a Mac, in a Sandboxed Apple Store app, are bookmarks fully supported ?

**Comment by maxkatz6** [comment]
It is supported in 11.2.0-beta.

**Reply by HEIC-to-JPEG-Dev** [reply]
Thank you
